horizon/pdp.py

```diff
@@ -283,6 +284,16 @@ def _configure_opal_data_updater(self):
             wait_time=1,
         )
 
+    def _configure_opal_offline_mode(self):
+        """
+        configure opal to use offline mode when enabled
+        """
+        opal_client_config.OFFLINE_MODE_ENABLED = sidecar_config.ENABLE_OFFLINE_MODE
+        opal_client_config.STORE_BACKUP_PATH = os.path.join(
+            os.path.dirname(sidecar_config.OFFLINE_MODE_BACKUP_PATH),
+            "policy_store_data.json",
+        )
```

Comment:
IMO this env var in OPAL should also be the directory and not the file itself

Reply:
Yeah I see what you mean (also relates to your earlier comments), When I've implemented offline mode for OPAL - the thought was that maybe the same volume gonna be mapped to the same directory for couple of OPAL instances, so maybe you'll want to differentiate the names.

Maybe I can have the directory (for both PDP & opal backups) as an ever + the file name as another envar (in case someone wants to control the later and not just the former). WDYT?

Reply:
sounds good to me